matlab/test/arrow/type/tDateTypeTest.m

```diff
@@ -0,0 +1,80 @@
+% Shared superclass for DateType-related tests for Date32Type and Date64Type
+classdef tDateType < matlab.unittest.TestCase
+    
+    properties
+        ConstructionFcn
+        ArrowType
+        TypeID
+        BitWidth
+        ClassName
+    end
+    
+    methods (Test)
+        function TestClass(testCase)
```

Comment:
Also, if you modify this class to inherit from `hFixedWidthType`, you won't need to redefine this test case.